qualification/Jenkinsfile

```diff
@@ -74,7 +74,8 @@ pipeline {
            sh 'pip install -r qualification/requirements.txt'
            sh 'pip install --no-deps ".[qualification]" && pip check'
            sh 'spead2_net_raw pytest -v -c qualification/pytest-jenkins.ini qualification --image-override katgpucbf:harbor.sdp.kat.ac.za/cbf/katgpucbf:latest'
-           sh 'qualification/report/generate_pdf.py report.json report.pdf'
+           COMMIT_ID = sh(script: 'qualification/report/generate_pdf.py report.json report.pdf -c', returnStdout: true)
+           currentBuild.displayName = "#${BUILD_NUMBER} ($COMMIT_ID)"
```

Comment:
If the qualification tests fail, does this line get run?

Reply:
No, and the report.json does not get published.

Reply:
I added an exception to handle pytest errors/failures and if json test reports are not generated.